code/src/main/java/com/onlinecontacttracing/storage/data-storage/Place.java

```diff
@@ -39,6 +39,10 @@ public long getIntervalEndSeconds() {
     return timeInterval.intervalEndSeconds;
   }
 
+  public String displayTimeIntervalAsDate(int timeZoneOffsetMinutes) {
```

Comment:
Java Time has a TimeZone class - let's pass that value around instead :) 

https://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html

Reply:
No ZoneOffset?